src/lifecycle.mjs

```diff
@@ -87,72 +77,24 @@ export async function setupFinder() {
   };
 }
 
-export function check(message) {
-  const valid = validate(message);
```

Comment:
Validation of messages must happen

Reply:
Yes, we also need validation of crawlPath messages once it's schema has been finalised. https://github.com/neume-network/message-schema/issues/19